### PR TITLE
docs: Add a write up to README about testing the federated schema

### PR DESCRIPTION
**Pull request by daemonsy**
See #74 for discussion

### CODE REVIEW

**Comment by daemonsy** [comment]
Hi @grxy , looks the the CI is hung. Any chance you can take a look at this and also review the PR ? :)

File: README.md [hunk 1]
```diff
@@ -176,6 +176,68 @@ To support [federated tracing](https://www.apollographql.com/docs/apollo-server/
     end
     ```
 
+## Testing the federated schema
+This library does not include any testing helpers currently. A federated service receives subgraph queries from the Apollo Gateway via the `_entities` field and that can be tested in a request spec.
+
+With apollo gateway setup to hit the service locally (or from query logs), you could retrieve `_entities` queries generated in the use of your application and test them.
```
Comment (toreriklinnerud) [inline README.md:182]:
```suggestion
With Apollo Gateway setup to hit the service locally (or from query logs), you could retrieve `_entities` queries generated in the use of your application and test them.
```

**Review by toreriklinnerud — APPROVED** [review]
The rspec example is very helpful 👍

**Review by geshwho — APPROVED** [review]
Thanks!

**Pull request merged**